src/button/_button.scss

```diff
@@ -31,6 +33,14 @@
   cursor: not-allowed;
 }
 
+.button__spinner-container {
+  position: absolute;
```

Comment:
I'm not sure but probably we need to set `top,left,right,bottom to 0` make sure it's on correct position